src/gamma.jl

```diff
@@ -627,10 +650,7 @@ function logabsgamma(x::Float32)
     y = ccall((:lgammaf_r,libopenlibm),  Float32, (Float32, Ptr{Int32}), x, signp)
     return y, Int(signp[])
 end
-logabsgamma(x::Real) = logabsgamma(float(x))
 logabsgamma(x::Float16) = Float16.(logabsgamma(Float32(x)))
```

Comment:
It seems wrong to convert both outputs:
```suggestion
function logabsgamma(x::Float16)
    y, s = logabsgamma(Float32(x))
    return Float16(y), s
end    
```